### DISCUSSION TITLE
Send cover (artwork)

### DISCUSSION THREAD

**Discussion by cybercuate** [savonet/liquidsoap#1784] [Q&A]
I know it is posible to show the artwork of each song playing in shoutcast transcoder v2. Is it possible in liquidsoap? what do I need to do to make it possible in my server?

**Comment by smimram** [comment]
I am not very familiar with shoutcast but I guess it is using the standard tag for cover art in mp3 files. If yours have such tags (commonly named `APIC`), you can try exporting them by adding `"apic"` to the list of exported tags, ie
```
set("encoder.encoder.export", ["artist", "title", "album", "genre", "date", "tracknumber", "comment", "track", "year", "dj", "next", "apic"])
```

BTW, I added this to the default settings in 65c3685886bd62b0e1124d4b037f8d1ee73cffb4.
